Ownership/Non-UK/Shannon.txt

```diff
@@ -15,7 +15,7 @@ OWNER:SNL:SN
 BORDER:SN_CTR03
 
 SECTOR:EISN2:24500:66000
-OWNER:SN:SNS:EI
+OWNER:SN:EI
```

Comment:
So there are no standard published or defined Shannon splits then?

Reply:
Correct, Shanon regularly changes sector boundaries and often without warning.